tutorials/gui/gui_skinning.rst

```diff
@@ -43,6 +43,11 @@ hierarchy, towards the root. If nothing was found, the default theme is
 used. This system allows for flexible overriding of themes in complex
 user interfaces.
 
+.. attention::
+   Do not use the custom theme option in the project settings, as there
+   are bugs with the theme propagation. Instead apply your theme to the
+   root control!
```

Comment:
```suggestion
.. attention::

    Don't use the custom theme option in the Project Settings, as there
    are known bugs with theme propagation. Instead, apply your theme to the
    root Control node's Theme property. It will propagate to instanced scenes
    automatically. To get correct theming in the editor for instanced scenes,
    you can apply the theme resource to the instanced scene's root node as well.
```